index.html

```diff
@@ -1205,6 +1205,90 @@ <h3>
           this risk.
         </p>
       </section>
+      <section>
+        <h3>
+          Fingerprinting and Data Leakage
+        </h3>
+        <h4>
+          Browser fingerprinting
+        </h4>
+        <p>
+          While the API ensures that no user data is ever shared without a
+          permission prompt, the longevity and uniqueness of real world
+          identifiers that are likely to be returned by the Digital Credentials
+          API make it a potential target for trackers and fingerprinters.
+        </p>
+        <p>
+          Even with selective disclosure, attackers might combine data from a
+          digital credential (such as the user's age, or the credential issuer,
+          timestamps, see [[[#leaking-incidental-data]]]) to reidentify and/or
+          fingerprint users.
+        </p>
+        <p>
+          This attack might be harder for third-party attackers (such as
+          scripts embedded on the verifier's pages, but not actively
+          collaborating with them for the purpose of tracking) because response
+          encryption is mandatory and responses should be decrypted on the
+          verifier's server. There may be some decrypted results not reflected
+          back to client-side JavaScript.
+        </p>
+        <p>
+          To further help users understand the full consequences of sharing
+          credentials with verifiers, user agents could add additional context
+          to their permission prompts, for example informing users who recently
+          cleared their cookies, or are using a "private browsing" mode, that
+          credentials can be used to reidentify them on the page.
+        </p>
+        <h4 id="leaking-incidental-data">
+          Leaking incidental data with credential presentations
+        </h4>
+        <p>
+          To ensure authenticity of a credential, its presentation to verifiers
+          generally includes more information than the content the verifier is
+          requesting access to. It will usually contain at least a signature of
+          the issuer and the wallet, and potentially other metadata.
+        </p>
+        <p>
+          This additional information could be used to reidentify and
+          fingerprint users, which is especially relevant when an otherwise
+          unlinkable presentation is made.
+        </p>
+        <p>
+          While the Digital Credentials API does not control the content of a
+          credential response, user agents can help protect users against this
+          type of tracking through clearly highlighting which information
+          likely gets shared with the verifier beyond what was requested, and,
+          more broadly, by identifying and blocking abuse of the API by
+          verifiers.
+        </p>
+        <h4>
+          Revealing device properties through feature and protocol availability
+        </h4>
+        <p class="issue" data-number="219">
```

Comment:
Note from @marcoscaceres: Let's get this written up in this PR as #221 will merge soon.